### PR TITLE
gdbus-codegen-glibmm: Split recipe into .inc and .bb file

### PR DESCRIPTION
**Pull request by thenor**
Change to use both a .inc and a .bb file. This enables projects to use
more than one version of the tool.

Signed-off-by: Therese Nordqvist <therese.nordqvist@pelagicore.com>

### CODE REVIEW

**Comment by thenor** [comment]
> split recipe into..?

Sounds better, yes. I was struggling to find a good way to express it.

File: recipes-devtools/gdbus-codegen-glibmm/gdbus-codegen-glibmm.inc [hunk 1]
```diff
@@ -0,0 +1,26 @@
+SUMMARY = "This is a code generator for generating D-Bus stubs and proxies from XML introspection files"
+DESCRIPTION = "This is a code generator for generating D-Bus stubs and proxies \
+from XML introspection files. The generated stubs and proxies are implemented \
+using glibmm and giomm. This generator is based on the gdbus-codegen code \
+generator which ships with glib."
+HOMEPAGE = "https://github.com/Pelagicore/gdbus-codegen-glibmm"
+LICENSE = "LGPLv2.1"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=4fbd65380cdd255951079008b364516c"
+PV = "1.0+git${SRCPV}"
```
Comment (thenor) [inline recipes-devtools/gdbus-codegen-glibmm/gdbus-codegen-glibmm.inc]: @martin-ejdestig Do you understand if this should be set to another version, e.g. 2.99 or 3.0?

Comment (sashko) [reply recipes-devtools/gdbus-codegen-glibmm/gdbus-codegen-glibmm.inc]: it'll be set by bitbake as you've version in the recipe name, so just remove this line

Comment (martin-ejdestig) [reply recipes-devtools/gdbus-codegen-glibmm/gdbus-codegen-glibmm.inc]: I guess this should not be set in the .inc file. And it does not need to be set in _X.Y.bb. So just remove it completely, I think. (If we do not want "+git<hash>" in the version.)

File: recipes-devtools/gdbus-codegen-glibmm/gdbus-codegen-glibmm_2.99.0.bb [hunk 2]
```diff
@@ -1,27 +1,5 @@
...
-}
+require gdbus-codegen-glibmm.inc
 
-# Dependencies for building this software.
-RDEPENDS_${PN} = "python3 python3-setuptools python3-jinja2"
-
-BBCLASSEXTEND = "native nativesdk"
-
-PACKAGE =+ "${PN}"
+SRC_URI = "git://github.com/pelagicore/gdbus-codegen-glibmm;branch=master"
```
Comment (sashko) [inline recipes-devtools/gdbus-codegen-glibmm/gdbus-codegen-glibmm_2.99.0.bb]: https? :)

File: recipes-devtools/gdbus-codegen-glibmm/gdbus-codegen-glibmm.inc (hunk 1 above)
Comment (thenor) [reply recipes-devtools/gdbus-codegen-glibmm/gdbus-codegen-glibmm.inc]: Alright.

**Pull request merged**